.github/workflows/tests.yml

```diff
@@ -52,20 +52,27 @@ jobs:
       - name: Install gauge (windows)
         if: matrix.os == 'windows-latest'
         run: |
-          git clone https://github.com/getgauge/gauge
+          git clone --depth=1 --branch=plugin-grpc https://github.com/getgauge/gauge
           cd gauge
           go run build/make.go --verbose
           go run build/make.go --install
           echo "::add-path::C:\Program Files\gauge\bin"
 
+      - name: Install html report
+        run: |
+          git clone --depth=1 https://github.com/getgauge/html-report
+          cd html-report
+          go run build/make.go
+          go run build/make.go --install
+
       - name: Install Gauge Java plugin from source
         run: |
           go run make.go
           go run make.go --install
 
       - name: Clone gauge-tests
         run: |
-          git clone --depth=1 https://github.com/getgauge/gauge-tests
+          git clone --depth=1 --branch=fileBasedScreenshot https://github.com/getgauge/gauge-tests
```

Comment:
The tests should also be cloned from master branch

Reply:
@negiDharmendra the tests are still cloned from a file.

Reply:
We can not clone from the `master`branch as there are some changes related to html-report FTs in the `fileBasedScreenshot` branch. Those changes can not be pushed to master until this [issue](https://github.com/getgauge/gauge/issues/1476) is addressed by all the runner.